UPGRADE-3.1.md

```diff
@@ -107,7 +107,7 @@ FrameworkBundle
    framework:
        cache:
            pools:
-               serializer:
+               cache.serializer:
```

Comment:
And by the way, adding the `cache.` prefix in the extension when registering the definition will also prevent us from potentially having conflicting service names because when people configure their own cache pools, they will not always think of adding the prefix here explicitly.